### PR TITLE
style: format files with black and fix warnings

### PR DESCRIPTION
**Pull request by subpop**
Format files using black, fix legitimate linting warnings, ignoring a couple that we regularly violate. This PR also includes a commit that refactors the `Makefile`. The commit drops the phony targets "install-lib-dir" and "dev-lib-dir", replacing them with a more traditional "in-template" target. This commit is included in this PR because the formatting attempted to format `constants.py`. Subsequently, when the makefile was invoked, the "install-lib-dir" target would further modify `constants.py`, breaking it, and consequently breaking the build.

### CODE REVIEW

**Review by DuckBoss — APPROVED** [review]
This looks okay to me, hopefully not missing anything :+1:

File: rhc_worker_playbook/constants.py [hunk 1]
```diff
@@ -1,8 +0,0 @@
-import os
-WORKER_LIB_DIR = os.path.join(os.path.dirname(__file__), "contrib")
-CONFIG_FILE = os.path.join(os.path.dirname(os.path.dirname(__file__)), "rhc-worker-playbook.toml")
-ANSIBLE_COLLECTIONS_PATHS = "/usr/share/rhc-worker-playbook/ansible/collections/ansible_collections/"
-# The path to the directory where artifacts should live.
-RUNNER_ARTIFACTS_DIR = "/var/log/rhc-worker-playbook/ansible"
-# Number of artifact directories we want to keep at most.
-RUNNER_ROTATE_ARTIFACTS = 100
```
Comment (m-horky) [inline rhc_worker_playbook/constants.py:8]: Is it necessary to lift out hardcoded paths into different place? I don't like projects where you have to `make` to run interpreted language.

File: .github/workflows/stylish.yml [hunk 2]
```diff
@@ -24,10 +24,11 @@ jobs:
...
         version: "24.3.0"
+        options: "--extend-exclude=yggdrasil_pb2.*.py"
 
     - name: Setup flake8 annotations
       uses: rbialon/flake8-annotations@v1
 
     - name: Run flake8
       run: |
-        flake8
+        flake8 --exclude=yggdrasil_pb2_grpc.py,yggdrasil_pb2.py --ignore=E501,E402
```
[the remark this answers is not in the document]
Comment (subpop) [reply .github/workflows/stylish.yml:34]: Sure. Is that a thing? I don't know much about flake8.

[the remark this answers is not in the document]
Comment (subpop) [reply .github/workflows/stylish.yml:34]: https://github.com/RedHatInsights/rhc-worker-playbook/pull/32

**Pull request merged**